ts/Series/Flags/FlagsSeries.ts

```diff
@@ -146,7 +147,9 @@ class FlagsSeries extends ColumnSeries {
             optionsY = options.y,
             yAxis = series.yAxis,
             boxesMap: Record<string, DistributedBoxObject> = {},
-            boxes: Array<DistributedBoxObject> = [];
+            boxes: Array<DistributedBoxObject> = [],
+            borderRadius = isNumber(options.borderRadius) ?
```

Comment:
Since `borderRadius` is defaulted to `0`, do we have to check if it's a number? Usually, we don't validate types for other options

Reply:
We need to do this because the upstream Series can take a configuration object for `borderRadius`. Alternatively we could resolve the config object for the flag series too, like we do in [some other series types](https://github.com/highcharts/highcharts/blob/747a438e6aed85f5e02c812521ebc584399ef56b/ts/Series/Sankey/SankeySeries.ts#L665-L669).

Reply:
Oh, okay, thanks for the explanation!

Let's keep it simple for now